mne/preprocessing/ica.py

```diff
@@ -2017,6 +2021,24 @@ def save(self, fname, *, overwrite=False, verbose=None):
             _write_ica(fid, self)
         return self
 
+    @property
+    def explained_variance_ratio_(self):
+        """The relative variance explained by the ICA components.
+
+        Returns
+        -------
+        array, shape (n_ica_components,) | None
+            Relative explained variance for each ICA component. If ICA has
+            not been fit, returns ``None``.
+        """
+        if self.current_fit == 'unfitted':
+            return None
```

Comment:
Although I agree with the instinct to use `None` here, I wonder if it might be more user-friendly to use something that has a better `__repr__`?  This may be overkill, but I'm imagining a class whose only purpose is to behave like `None` except for when printed, it will say "explained variance ratio unavailable (model not yet fit)" or some similarly clear message.


...thinking further, maybe this is good enough:

```suggestion
        if self.current_fit == 'unfitted':
            logger.info('Explained variance ratio is None (model not yet fitted)')
            return None
```

Reply:
I like the simple approach you suggested here, let's start with this first!

Reply:
I have updated the output produced by the MWE in the top comment here to include the log message that is now being produced.